stablehlo/dialect/Serialization.cpp

```diff
@@ -79,5 +79,7 @@ OwningOpRef<ModuleOp> deserializePortableArtifact(StringRef sourceStr,
   return module;
 }
 
+StringRef getEarliestForwardCompatibleVersion() { return "0.9.0"; }
```

Comment:
Should we also add `getEarliestBackwardCompatibleVersion()`, given that these two don't have to be the same?

Reply:
Should we wait for a use case for that? Given that there is no way to tell what version a payload is currently, I'm not sure this method is very useful.